src/tasks/nli_probing.py

```diff
@@ -50,12 +51,17 @@ def __init__(self, path, max_seq_len, name, probe_path="probe_dummy.tsv",
 
     def load_data(self, path, max_seq_len, probe_path):
         targ_map = {'neutral': 0, 'entailment': 1, 'contradiction': 2}
-        tr_data = load_tsv(os.path.join(path, 'train_dummy.tsv'), max_seq_len,
-                           s1_idx=1, s2_idx=2, targ_idx=None, skip_rows=0)
-        val_data = load_tsv(os.path.join(path, 'lexnegs.tsv'), max_seq_len,
-                            s1_idx=8, s2_idx=9, targ_idx=10, targ_map=targ_map, skip_rows=1)
-        te_data = load_tsv(os.path.join(path, 'test_dummy.tsv'), max_seq_len,
-                           s1_idx=1, s2_idx=2, targ_idx=None, skip_rows=0)
+        label_fn = targ_map.__getitem__
+        tr_data = load_tsv(data_file=os.path.join(path, 'train_dummy.tsv'), max_seq_len=max_seq_len,
+                           s1_idx=1, s2_idx=2, label_idx=None, skip_rows=0,
+                           tokenizer_name=self._tokenizer_name)
+        # TODO(?): should this use probe_path?
```

Comment:
@najoungkim should these use `probe_path`? It's passed into the function but not used.